apis/r/R/TileDBGroup.R

```diff
@@ -365,21 +388,27 @@ TileDBGroup <- R6::R6Class(
     # list with names: name, uri, and type.
     get_all_members_uncached_read = function(group_handle) {
 
-      count <- tiledb::tiledb_group_member_count(group_handle)
+      #count <- tiledb::tiledb_group_member_count(group_handle)
+      count <- c_group_member_count(group_handle)
```

Comment:
Can we clean out the commented code?
```suggestion
      count <- c_group_member_count(group_handle)
```